### ISSUE TITLE
[Facebook Messenger] "Keep the conversation going from your phone" banner 

### ISSUE THREAD

**Issue by Betree** [saenzramiro/rambox#278]
Problem most certainly come from Facebook which seems to ignore Linux user agent as this also displays in my chromium. I've sent a bug report to Facebook but it seems this has been around for a while as Facebook-Messenger-Desktop project has worked on it in April 2015 : https://github.com/Aluxian/Facebook-Messenger-Desktop/issues/18.

It looks like that :
![image](https://cloud.githubusercontent.com/assets/1556356/18071128/5ab70c24-6e53-11e6-8f38-b237a7ce2871.png)

## 

> Rambox 0.4.1
> Electron 1.3.1
> linux x64 4.4.0-22-generic

**Comment by Prehistoricman** [comment]
I also have this issue on all themes. Looking at the dev tools, it appears to be inside the object for the whole window. Maybe CSS would work to keep it hidden?

**Comment by saenzramiro** [comment]
@Prehistoricman Yes, that's why I add this to milestone 1.0.0. In that release I pretend to add custom CSS code to services.

**Comment by saenzramiro** [comment]
BTW, that message shows because you never installed the app in your phone. If you do (and then uninstall it), that message will disappear.

**Comment by jawtheshark** [comment]
@saenzramiro That's all find and dandy, but what about the people who do not have a smartphone? 
Besides, I used to have the FB App installed, and I don't want it.  I went completely browser based for FB and it works just fine.  Yet, Facebook keeps on nagging me to install their crap.  

I guess "No", doesn't mean "No", right?

**Comment by saenzramiro** [comment]
This will be fixed in a future release when we add Custom CSS code injection.
